lms/templates/sysadmin_dashboard_gitlogs.html

```diff
@@ -112,13 +114,13 @@ <h2 class="instructor-nav">
                 <td>${cil.git_log}</td>
               </tr>
           %endfor
-          ## Show the full log if viewing information about a specific course
-          %if course_id is not None:
+          ## Show the full log of the latest import if viewing information about a specific course
+          %if detailed_view:
               <tr>
                 <td colspan="3">
                   <pre>
-                    %if cil is not None:
-                        ${cil.import_log | h}
+                    %if logs[0] is not None:
```

Comment:
I'm pretty sure this is the cause of your test failure, you still need to check if `cil` exists.